frontend/app/custom_filters.py

```diff
@@ -78,12 +83,30 @@ def _jinja2_filter_datetime(date, fmt=None):
     return native.strftime(format)
 
 
-def cgmlst_count_called(alleles):
-    return sum(1 for allele in alleles.values() if allele is not None)
+def cgmlst_count_called(alleles: Dict[str, int | str | None]) -> int:
+    """Return the number of called allleles
+
+    :param alleles: called allleles
```

Comment:
```suggestion
def cgmlst_count_called(alleles: Dict[str, int | str | None]) -> int:
    """Return the number of called alleles
    
    Strings and null values are treated as failed calls and are not counted.

    :param alleles: called alleles
```

Small suggestion, since it was explicitly stated in the `cgmlst_count_missing` function below. + Corrected some typos.